packages/core/package.json

```
@@ -25,7 +25,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "",
+    "url": "https://github.com/VKCOM/mini-apps-analytics/tree/master/packages/core",
```

```suggestion
    "url": "https://github.com/VKCOM/mini-apps-analytics.git",
```

в **directory** уже указано `packages/core`

---

спасибо. поправил